File: src/3238.find-the-number-of-winning-players.3519/solution.ts
function winningPlayerCount (n: number, pick: number[][]): number {
    const counts = new Array<number[]>(n);
    for (let i = 0; i < n; i++) {
        counts[i] = new Array<number>(11).fill(0);
    }
    for (const [x, y, ] of pick) {
        counts[x][y]++;
    }
    let result = 0;
    for (let i = 0; i < n; i++) {
        let flag = false;
        for (const c of counts[i]) {
            if (c > i) {
                flag = true;
            }
        }
        if (flag) {
            result++;
        }
    }
    return result;
}

File: src/3238.find-the-number-of-winning-players.3519/solution.java
class Solution {
    public int winningPlayerCount(int n, int[][] pick) {
        int[][] counts = new int[n][11];
        for(int[] row : pick){
            counts[row[0]][row[1]]++;
        }

        int result = 0;
        for(int i=0;i<n;i++){
            for(int j=0;j<11;j++){
                if(counts[i][j]>i){
                    result++;
                    break;
                }
            }
        }
        
        return result;
    }
}